pkg/addons/manifest.go

```diff
@@ -128,6 +133,17 @@ func (a *applier) loadAddonsManifests(
 			tplDataParams[k] = v
 		}
 
+		// Resolve environment variables in Params
+		for k, v := range tplDataParams {
+			if strings.HasPrefix(v, paramsEnvPrefix) {
+				envName := strings.TrimPrefix(v, paramsEnvPrefix)
+				if env := os.Getenv(envName); len(env) > 0 {
```

Comment:
I think an empty env is okay and we should respect it. Therefore, I'd replace this with [`os.LookupEnv`](https://pkg.go.dev/os#LookupEnv):

```suggestion
				if env, ok := os.Getenv(envName); ok {
```

Also, we should consider returning an error if the provided environment variable doesn't exist. I think that would provide a better UX instead of silently failing.

Reply:
Fair point, I'll change that!